test/haskell/default_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Definition specific tests
+EXPECTED_VERSION=$(~/.ghcup/bin/ghcup list | grep "ghc " | grep recommended | awk '{print $3}')
```

Comment:
```suggestion
EXPECTED_VERSION=$(${_REMOTE_USER_HOME}/.ghcup/bin/ghcup list | grep "ghc " | grep recommended | awk '{print $3}')
```

For the failing MCR tests https://github.com/devcontainers/features/actions/runs/4369291649/jobs/7642925890#step:4:358

I think, ghcup is getting installed at `/root/*` but is the tests are checking at `/home/vscode/*`
The suggestion should help.

Reply:
Ooops wrong place!